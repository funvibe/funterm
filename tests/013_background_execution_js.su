js (background_task) {
    function background_task() {
        const start = Date.now();
        while (Date.now() - start < 200) {
            // Busy wait for 200ms
        }
        console.log("Background task completed");
    }
}
    
js.background_task() &
lua.print("After background task start")