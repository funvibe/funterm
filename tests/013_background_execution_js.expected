After background task start
Background task completed